.eslintrc.yml

```diff
@@ -1,5 +1,4 @@
 extends:
-  - standard-jsx
```

Comment:
Can you provide more detail on why this ruleset and associated dependencies are being removed?

Reply:
`eslint-config-standard-react` extends `eslint-config-standard-jsx` https://github.com/standard/eslint-config-standard-react/blob/master/eslintrc.json#L21 so I think it's not justified to add it separately.